otter/convergence/planning.py

```diff
@@ -84,7 +84,11 @@ def _remove_from_lb_with_draining(timeout, nodes, now):
 
     changes = [drain_lb_node(node=node) for node in to_drain]
 
-    return removes + changes
+    retry = (
+        [ConvergeLater(reasons=[ErrorReason.String('draining servers')])]
+        if in_drain else [])
```

Comment:
Sounds like a good idea. However I am worried the number of steps will be too much when there are many servers. I think we can leave it like this and implement this or add separate log inside `log_steps` if such a need arises.